java/src/jmri/jmrix/AbstractSerialPortController.java

```diff
@@ -84,23 +84,60 @@ public void connect() throws java.io.IOException {
     /**
      * Do the formal opening of the port,
      * set the port for blocking reads without timeout,
-     * and set the port to 8-2-none.
+     * set the port to 8-1-none,
```

Comment:
No need to change, but for someone who doesn't know much about serial communication, it would be better if this comment was:

* set the port to 8 bits, 1 stop bit, no parity.

Reply:
Good point, thanks